eng/scripts/Get-TypeSpec-Folders.ps1

```diff
@@ -12,7 +12,8 @@ $changedFiles = @()
 $allChangedFiles = (Get-ChildItem -path ./specification tspconfig.yaml -Recurse).Directory.FullName | ForEach-Object {[IO.Path]::GetRelativePath($($pwd.path), $_)}
 $allChangedFiles = $allChangedFiles -replace '\\', '/'
 
-if ([string]::IsNullOrEmpty($TargetBranch) -or [string]::IsNullOrEmpty($SourceBranch)) {
+$ci = [string]::IsNullOrEmpty($TargetBranch) -or [string]::IsNullOrEmpty($SourceBranch)
```

Comment:
What if someone runs this locally with an invalid target or source? It wouldn't be a CI run. 

The common way we detect running in a DevOps job is by looking this env variable https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/logging.ps1#L3

Reply:
we also want locally runs of "run all" to skip these files as well. but a "runall" variable might be confusing between ci/runall option and runall due to eng and root dir